src/save/encodeBase64.ts

```diff
@@ -0,0 +1,18 @@
+import { toBase64 } from '@jsonjoy.com/base64';
+
+import type { Image } from '../Image.js';
+
+import { encode } from './encode.js';
+import type { ImageFormat } from './encode.js';
+/**
+ * Converts image into a base64 URL string.
+ * @param image - Image to get base64 encoding from.
+ * @param format - Image format.
+ * @returns base64 string.
+ */
+export function encodeBase64(image: Image, format: ImageFormat) {
```

Comment:
```suggestion
export function encodeDataURL(image: Image, format: ImageFormat) {
```

And rename the file, tests, etc.